### PR TITLE
chore: add script to get ui-components bundle size

### PR DESCRIPTION
**Pull request by nelsonmestevao**
### Output preview

image

### References

https://stenciljs.com/docs/config#sourcemap
https://github.com/danvk/source-map-explorer/blob/master/README.md#generating-source-maps
https://github.com/ionic-team/stencil/issues/1744

### CODE REVIEW

File: packages/ui-components/stencil.config.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export const config: Config = {
 		}
 	],
 	plugins: [sass()],
+	sourceMap: true,
```
Comment (miguelpinto98) [inline packages/ui-components/stencil.config.ts:54]: we should only generate sourceMaps for development purposes.

**Review by miguelpinto98 — CHANGES_REQUESTED** [review]

File: packages/ui-components/stencil.config.ts (hunk 1 above)
Comment (nelsonmestevao) [reply packages/ui-components/stencil.config.ts]: Do you think [this](https://github.com/kelvininc/ui-components/pull/204/commits/507a9bda0e77df01388ba32a0f68443e61087fa6) is a good solution?

**Comment by nelsonmestevao** [comment]
> Nice +1
> 
> Could you add a screenshot of the `source-map-explorer` output? Thank you +1

Thanks :pray: Checkout the description now :raised_hands:

**Review by GoncaloCamaz — APPROVED** [review]

File: packages/ui-components/stencil.config.ts (hunk 1 above)
Comment (miguelpinto98) [reply packages/ui-components/stencil.config.ts]: Hmmm, is it possible to use the environment variables to generate the source maps? We can have 2 commands as you do but with the same configuration (conditional to the environment).

Comment (nelsonmestevao) [reply packages/ui-components/stencil.config.ts]: I tried to find a way to use the `--dev` for generating the source maps but I didn't find a way in the documentation to create conditional configuration target. Do you know how?

**Review by miguelpinto98 — APPROVED** [review]
🚀

**Comment by miguelpinto98** [comment]
Seems good, squash all commits to merge with dev @nelsonmestevao

**Pull request merged**